code/game/objects/items/devices/last_shelter.dm

```diff
@@ -2,7 +2,7 @@ GLOBAL_DATUM(last_shelter, /obj/item/device/last_shelter)
 
 /obj/item/device/last_shelter
 	name = "Last Shelter"
-	desc = "Powerful scanner that can teleport a cruciforms of pilgrims lost in this sector of space."
+	desc = "A sacred Mekhanite artifact that passively scans nearby sectors of space and pre-emptively recovers cruciforms when they are at risk of being lost in space. Also functions to summon lost souls when activated." // OCCULUS EDIT - More accurate description
```

Comment:
"A sacred Mekhanite artifact that passively scans the current sector of space. It recovers the cruciforms of the faithful who are lost to the void."